pathfinder/token_network.py

```diff
@@ -196,18 +197,22 @@ def update_fee(
         channel_view.fee = new_fee
 
     def get_paths(
-        self,
-        source: Address,
-        target: Address,
-        value: int,
-        k: int,
-        **kwargs
+            self,
+            source: Address,
+            target: Address,
+            value: int,
+            k: int,
+            **kwargs
     ):
         visited: Dict[ChannelId, float] = {}
         paths = []
         hop_bias = kwargs.get('bias', 0)
 
-        def weight(u: Address, v: Address, attr: Dict[str, Any]):
+        def weight(
+                u: Address,
+                v: Address,
+                attr: Dict[str, Any]
+        ):
             view: ChannelView = attr['view']
             assert 0 <= hop_bias <= 1
```

Comment:
Move the `hop_bias` check out of the `weight` function. It's constant for a single `get_paths` call so we don't need to check it on each edge traversal.